<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-BI4283QF" failures="3" tests="6" name="Test1" time="111.256" errors="0" timestamp="2020-03-27T18:10:00 CDT">
  <testcase classname="com.nopcommerce.testcases.TC_Logintest_001" name="Logintest" time="17.088"/>
  <testcase classname="com.nopcommerce.testcases.TC_LoginDDT_002" name="loginTest" time="12.615">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.nopcommerce.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.nopcommerce.testcases.TC_LoginDDT_002" name="loginTest" time="4.267">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.nopcommerce.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.nopcommerce.testcases.TC_LoginDDT_002" name="loginTest" time="4.051">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at com.nopcommerce.testcases.TC_LoginDDT_002.loginTest(TC_LoginDDT_002.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.nopcommerce.testcases.TC_LoginDDT_002" name="loginTest" time="11.656"/>
  <testcase classname="com.nopcommerce.testcases.TC_AddCustomer_003" name="addnewcustomer" time="34.926"/>
</testsuite> <!-- Test1 -->
